AGENTS.md

```diff
@@ -0,0 +1,48 @@
+# Repository Guidelines
+
+This repo builds and tests synthetic reproductions of Agoric mainnet (agoric-3) proposals. It publishes Docker images that encode the chain state across proposals and verifies expected outcomes.
```

Comment:
This says nearly the same thing as the top of the README:

> This repo serves several functions:
> 
> - verify building an image with in which known proposals have executed
> - publishing an image with all passed proposals
> - verify that certain tests pass after each proposal

Is there a reason to not use the same copy in both places?

For that matter: Are agents not expected to look at README?

I could imagine they don't look at CONTRIBUTING, though they should. But are we really supposed to maintain 2 copies of basic README info?